docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '2.3'
```

Comment:
why is the version 2.3 ??

Reply:
Because it is the only one supported by Nvidia-Docker:
https://github.com/NVIDIA/nvidia-docker/wiki/Frequently-Asked-Questions#do-you-support-docker-compose

```
Do you support Docker Compose?
Yes, use Compose format 2.3 and add runtime: nvidia to your GPU service. Docker Compose must be version 1.19.0 or higher.
```

Reply:
I didn't know that it was the only one supported by Nvidia-Docker. now, this makes sense to me! Thanks! @enric1994